egs2/clarity/enh_2021/local/prep_data.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+import os
+import json
+from pathlib import Path
```

Comment:
Could you fix the import order to follow the alphabet order?

Reply:
done